.github/workflows/build_and_test.yml

```diff
@@ -579,7 +579,7 @@ jobs:
           --user "${{ secrets.CI_ACCESS_TOKEN }}" \
           --data \
             "{
-              \"ref\": \"main\",
+              \"ref\": \"sharnoff/vm-file-cache-on-disk\",
```

Comment:
leftover testing change?

Reply:
Intentionally for now, to run CI both with/without cloud using the new argument. Next commit will be removing this, assuming CI passes.